content/en/apps/guides/data/analytics/setup.md

```diff
@@ -0,0 +1,108 @@
+| `DBT_POSTGRES_HOST`       | `postgres`                                         | PostgreSQL instance IP or endpoint                                                                                                             |
+| `CHT_PIPELINE_BRANCH_URL` | `"https://github.com/medic/cht-pipeline.git#main"` |                                                                                                                                                |
+| `COUCHDB_USER`            | `medic`                                            | Username of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_PASSWORD`        | `password`                                         | Password of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_DBS`             | `"medic"`                                          | Space separated list of databases to sync e.g `"medic medic_sentinel"`                                                                         |
+| `COUCHDB_HOST`            | `couchdb`                                          | Host of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_PORT`            | `5984`                                             | Port of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_SECURE`          | `false`                                            | Is connection to CouchDB instance secure?                                                                                                      |
+
+{{% alert title="Note" %}}
+If `CHT_PIPELINE_BRANCH_URL` is pointing to a private repo then you need to provide an access token in the url i.e. `https://<PAT>@github.com/medic/cht-pipeline.git#main`. In this example you will replace `<PAT>`  with an access token from Github. Instruction on how to generate one can be found [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+{{% /alert %}}
+
+### Setup
+The following configurations facilitate data synchronization, transformation, and storage for local development and testing.
+
+Copy the values in `env.template` file to the `.env` file and update them accordingly to the local configuration for the different scenarios below.
+
+Install the dependencies:
+```sh
+npm install
+```
+
+#### With sample CouchDB data
+This setup involves starting Logstash, PostgreSQL, PostgREST, DBT, and CouchDB. Sample fake data is generated for CouchDB.
+
+Run the Docker containers and wait for every container to be up and running:
+```sh
+# starts logstash, postgres, postgrest, generator (for fake data), couchdb and dbt
+npm run local
```

Comment:
We just told them what this command will do, we can simplify by removing duplicate content:

```suggestion
npm run local
```

Also - is there a reason this command gets a special `npm` entry while the rest are all raw `docker compose` calls?  Maybe remove the use of `npm` here so there's parity among all the calls to all be `docker compose`?